File: PL024.cpp
#include <iostream>
using namespace std;

string compareNM(int n, int m)
{
    if (n < m)
    {
        return "lesser";
    }
    else if (n == m)
    {
        return "equal";
    }
    else
    {
        return "greater";
    }
}

int main()
{
    int n = 9, m = 3;
    cout << "The relationship between " << n << " and " << m << " is : " << n << " is " << compareNM(n, m) << " than " << m << "." << endl;

    return 0;
}

File: PL024.java
public class PL024 {
    public static void main(String[] args) {
        int n = 4, m = 3;
        System.out.println("The relationship between " + n + " and " + m + " is : " + n + " is " + compareNM(n, m)
                + " than " + m + ".");
    }

    public static String compareNM(int n, int m) {
        if (n < m) {
            return "lesser";
        } else if (n == m) {
            return "equal";
        } else {
            return "greater";
        }
    }
}
